### PR TITLE
Article on other WPILib versions

### PR DESCRIPTION
**Pull request by dylantknguyen**
Adds article in order to address #332

### CODE REVIEW

File: source/docs/getting-started/getting-started-frc-control-system/other-wpilib-versions.rst [hunk 1]
```diff
@@ -0,0 +1,76 @@
+Install Other WPILib Versions
+=============================
+
+This article contains instructions on building projects using a development build and a local WPILib build
```
Comment (prateekma) [inline source/docs/getting-started/getting-started-frc-control-system/other-wpilib-versions.rst:4]: Needs period at end of sentence.

Comment (dylantknguyen) [reply source/docs/getting-started/getting-started-frc-control-system/other-wpilib-versions.rst]: Just added.

File: source/docs/getting-started/getting-started-frc-control-system/other-wpilib-versions.rst [hunk 2]
```diff
@@ -1,7 +1,7 @@
 Install Other WPILib Versions
 =============================
 
-This article contains instructions on building projects using a development build and a local WPILib build
+This article contains instructions on building projects using a development build and a local WPILib build.
 .. note:: This only applies to C++/Java teams.
```
Comment (Daltz333) [inline source/docs/getting-started/getting-started-frc-control-system/other-wpilib-versions.rst:5]: Please have a newline separating the note from the initial article.

**Comment by Daltz333** [comment]
You'll need to run make translate.

File: source/docs/contributing/wpilib/other-wpilib-versions.rst [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+=============================
+
+This article contains instructions on building projects using a development build and a local WPILib build.
+
+.. note:: This only applies to Java/C++ teams.
+
+Development Build
+-----------------
+
+In order to build a project using a development build, find the ``build.gradle`` file and open it. Then, add the following code below the plugin section and replace ``YEAR`` with the year of the development version.
```
Comment (sciencewhiz) [inline source/docs/contributing/wpilib/other-wpilib-versions.rst:11]: I think this needs some introduction of what a development build is and what the caveats are, ie that every commit to master is built, and that sometimes (ie around beta) development builds won't work without the beta image.

**Pull request merged**